k8s/deployments/connectors-worker-notion-deployment.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: connectors-worker-notion-deployment
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: connectors-worker
+      worker: notion
+  template:
+    metadata:
+      labels:
+        app: connectors-worker
+        name: connectors-worker-pod
+        worker: notion
+        admission.datadoghq.com/enabled: "true"
+      annotations:
+        ad.datadoghq.com/web.logs: '[{"source": "connectors-worker","service": "connectors-worker","tags": ["env:prod"]}]'
+    spec:
+      containers:
+        - name: web
+          image: gcr.io/or1g1n-186209/connectors-image:latest
+          args: ["--workers", "$WORKERS"]
```

Comment:
Where is thaat env var coming from ?

Reply:
Defined right below, preparing our move to Terraform 😅.